eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala

```diff
@@ -162,6 +162,19 @@ class PeerSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with StateTe
     probe.expectMsg(PeerConnection.ConnectionResult.AlreadyConnected)
   }
 
+  test("handle unknown messages") { f =>
+    import f._
+
+    val listener = TestProbe()
+    system.eventStream.subscribe(listener.ref, classOf[UnknownMessageReceived])
+    connect(remoteNodeId, peer, peerConnection, channels = Set(ChannelCodecsSpec.normal))
+
+    peerConnection.send(peer, UnknownMessage(tag = 50001, data = ByteVector.empty))
```

Comment:
When someone reads this test out of context, they will have no way of understand why 50001 triggers an event whereas 60001 doesn't...

I think you should add a `val TestPluginParams = PluginParams(Set(50001), TestFeature)` inside `TestConstants`, and in this test you should do:

```
peerConnection.send(peer, UnknownMessage(tag = TestPluginParams.tags.head, data = ByteVector.empty))
```

You should also use different values for the `TestFeature` and the message tag, otherwise it's misleading. They have no reason to be the same (usually a feature will be associated with multiple messages). Set `TestFeature` to any random value you like.